### DISCUSSION TITLE
We are organizing our second Community Standup!

### DISCUSSION THREAD

**Discussion by vnbaaij** [microsoft/fluentui-blazor#2326] [Announcements]
We would like to get to know our contributors and users a bit better and are therefore organizing our second ever 'Fluent UI Blazor library Community Standup'.

We originally aimed for September 23rd but we are not going to make that. We're now looking at October 8. ~~A new registration form will be made available soon~~.

Sign up form is here: https://forms.office.com/r/HBsT8N0dCF

Looking forward to seeing you there (again)!

**Comment by MarvinKlein1508** [comment]
@vnbaaij does the invite get sent out automatically? I'm just wondering since I haven't received a confirmation yet.

**Reply by vnbaaij** [reply]
I'll send it out somewhere next week. Want to give people time to fill out the form and then do the send in one go. We have 21 participants so far!

**Reply by MarvinKlein1508** [reply]
Did you sent them out yet? I still have not received anything.

**Reply by vnbaaij** [reply]
~~Working on it right now~~ Sent!

**Reply by franklupo** [reply]
Hi, I didn't sign up for tomorrow because I'm not there. Is it possible to see a video of the event?

**Comment by vnbaaij** [comment]
Next one is preliminary scheduled for 17th of September!

**Reply by franklupo** [reply]
is there a video of the previous one?

**Reply by vnbaaij** [reply]
No sorry, we forgot to record.

**Reply by albx** [reply]
I missed the first but I'd like to join the second community standup. The form linked above is still valid?
Thank you

**Reply by vnbaaij** [reply]
No, we will put out a new form soon. We need to reschedule...

**Reply by albx** [reply]
Thank you for the reply :)

**Comment by sebitsi** [comment]
Hi.

Was this event recorded?
Where can we find it?

**Reply by vnbaaij** [reply]
No, we don't record these as people can be free to talk and don't have to be held back by language barrier or anything like that.

**Reply by MarvinKlein1508** [reply]
Do you already have plans for the second community stand-up?

**Reply by vnbaaij** [reply]
The second one took place on Oct 8.

We haven't scheduled the 3rd meeting yet.

**Reply by MarvinKlein1508** [reply]
Ah I've missed it because you only edited the last post. Would be great if you can post new announcements for the next ones.
